lib/xapi/implementation.rb

```diff
@@ -102,6 +104,13 @@ def readme_body
       ].reject(&:empty?).join("\n").strip
     end
 
+    def incomplete_solutions_body
+      <<-README
+## Submitting Incomplete Problems
+It's possible to submit an incomplete solution so you can see how others have completed the exercise.
+      README
+    end
```

Comment:
One of these things is not like the others.
Note that none of the other text sources are coming from a hardcoded string.